sui/open_rpc/samples/objects.json

```diff
@@ -6,24 +6,32 @@
         "dataType": "moveObject",
         "type": "0x2::DevNetNFT::DevNetNFT",
         "fields": {
-          "description": "An NFT created by the wallet Command Line Tool",
+          "description": {
+            "string": "An NFT created by the wallet Command Line Tool"
+          },
           "id": {
-            "id": "0x2d835f2e30e360c32318f55e5f33477004284b04",
-            "version": 1
+            "versionedID": {
+              "id": "0x9ee9e080b3b40f7ddaeb911f6fb0dc29613229b0",
+              "version": 1
+            }
+          },
+          "name": {
+            "string": "Example NFT"
           },
-          "name": "Example NFT",
-          "url": "ipfs://bafkreibngqhl3gaa7daob4i2vccziay2jjlp435cf66vhono7nrvww53ty"
+          "url": {
+            "string": "ipfs://bafkreibngqhl3gaa7daob4i2vccziay2jjlp435cf66vhono7nrvww53ty"
```

Comment:
One thing I don't like about this is that sometimes the field name refers to a type (e.g., `string`) while in other places it's not (e.g., version).

Reply:
I have removed the tag in the parent branch, I have condensed the enum types instead of using tags to remove ambiguity.